share/availability/discovery/discovery.go

```diff
@@ -57,6 +61,19 @@ func NewDiscovery(
 		peersLimit,
 		discInterval,
 		advertiseInterval,
+		nil,
+	}
+}
+
+func (d *Discovery) WithOnPeersUpdate(f OnPeersUpdate) {
```

Comment:
could you please add a comment, that the current option is extending(and not replacing) the default callback with the custom one?

Reply:
done